core/xml.js

```diff
@@ -92,29 +99,30 @@ Blockly.Xml.blockToDomWithXY = function(block, opt_noId) {
     }
   }
 
-  var width;  // Not used in LTR.
+  let width;  // Not used in LTR.
   if (block.workspace.RTL) {
     width = block.workspace.getWidth();
   }
 
-  var element = Blockly.Xml.blockToDom(block, opt_noId);
-  var xy = block.getRelativeToSurfaceXY();
-  element.setAttribute('x',
-      Math.round(block.workspace.RTL ? width - xy.x : xy.x));
+  const element = blockToDom(block, opt_noId);
+  const xy = block.getRelativeToSurfaceXY();
+  element.setAttribute(
+      'x', Math.round(block.workspace.RTL ? width - xy.x : xy.x));
   element.setAttribute('y', Math.round(xy.y));
   return element;
 };
+exports.blockToDomWithXY = blockToDomWithXY;
 
 /**
  * Encode a field as XML.
- * @param {!Blockly.Field} field The field to encode.
+ * @param {!Field} field The field to encode.
  * @return {?Element} XML element, or null if the field did not need to be
  *     serialized.
  * @private
  */
-Blockly.Xml.fieldToDom_ = function(field) {
+const fieldToDom_ = function(field) {
```

Comment:
Rename this functions to remove the underscore, and also remove the private tag, since they are now private by virtue of not being exported.